docs/source/articles/test-a-credit-card-transaction.html.md

```diff
@@ -1,16 +1,13 @@
 ---
 title: Test a Credit Card Transaction
-excerpt: Active Merchant provides the following credit card numbers for use in development environments.
+excerpt: How to use the default "bogus" gateway to test credit card transactions in development environments
+created_at: 2018/07/31
 ---
 
-# Test a Credit Card Transaction
-
-[Active Merchant](http://activemerchant.org/) provides the following credit card numbers for use in development environments.
-
-| Card Number | Result |
-| --- | --- |
-| 1 | Successful tansaction |
-| 2 | Declined transaction |
-| 3 | Payment processing error |
+Test a Credit Card Transaction
```

Comment:
Maybe include something like: "Some credit card processors only accept specific credit card numbers in the test environment. Be sure to check your processors documentation."

Reply:
I tried to make it more clear that this doc is referring to the behavior of the default bogus gateway only. This is a legacy doc that I thought was worth keeping, but cleaned it up a little.